### DISCUSSION TITLE
OMI glTF Working Group Meeting 2023-02-02

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#140] [General]
This meeting is on 1/26/2023 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:
* We discussed https://openmv.org (Open Metaverse Foundation) and how we can synergize with their efforts. Should we make a showcase of our projects to open up a broader discussion?
* jin - proposals how to make our specs and work more accessible to others. showcase gltf models
* Review pull requests https://github.com/omigroup/gltf-extensions/pulls
* [EXT_skin_humanoid](https://github.com/takahirox/EXT_skin_humanoid)
* [OMI_personality proposal #133](https://github.com/omigroup/gltf-extensions/pull/133) - may postpone because ant couldn't make it this week.
* [OMI_link](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_link)
* [OMI_spawn_point](https://github.com/omigroup/gltf-extensions/tree/main/extensions/2.0/OMI_spawn_point)
* Review [MSFT_physics](https://github.com/eoineoineoin/glTF_Physics)
  * [Replace `isKinematic` with a string enum for the body type eoineoineoin/glTF_Physics#4](https://github.com/eoineoineoin/glTF_Physics/issues/4)
  * [Remove the `centerOfMass` property eoineoineoin/glTF_Physics#5](https://github.com/eoineoineoin/glTF_Physics/issues/5)
  * [Remove the `isHollow` parameter from collider shapes eoineoineoin/glTF_Physics#6](https://github.com/eoineoineoin/glTF_Physics/issues/6)
  * [Define explicit units for all relevant properties eoineoineoin/glTF_Physics#7](https://github.com/eoineoineoin/glTF_Physics/issues/7)
* Licensing Extension https://twitter.com/superhoge/status/1596757861882740736
* [KHR_audio / Khronos feedback](https://github.com/KhronosGroup/glTF/pull/2137)
  
  * [gltf-sample-assets](https://github.com/KhronosGroup/glTF-Sample-Models/pull/360)
* [Third Room Extensions](https://github.com/matrix-org/thirdroom/pull/469)
* [glXF](https://github.com/KhronosGroup/glXF/blob/main/specification/2.0/README.md)
* Show and tell
* Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by robertlong** [comment]
# UnityGLTF
- Progress made on UnityGLTF conformance and extension API 
- Published PRs for those fixes https://github.com/prefrontalcortex/UnityGLTF/pulls

# Matrix Specs under development
- https://github.com/matrix-org/thirdroom/pull/469
- Implementations in Third Room Unity Exporter https://github.com/matrix-org/thirdroom-unity-exporter

# Lightmap Specs
- [MOZ_lightmap](https://github.com/takahirox/MOZ_lightmap)
- [MX_lightmap](https://github.com/matrix-org/thirdroom/blob/be326130382e7de514be504c6b658ae60bd2463c/docs/gltf-extensions/MX_lightmap/README.md)
- [needle_lightmaps](https://github.com/needle-tools/needle-engine-support/blob/main/documentation/technical-overview.md#needle_lightmaps)

**Reply by madjin** [reply]
I'm working on documentation and examples for glTF based lightmaps here (WIP):
- https://github.com/madjin/lightmap-tests
- https://hackmd.io/@XR/rkx6PV7nj

**Comment by robertlong** [comment]
# Cross Community Open Metaverse Expo
- Feb 17th and onwards
- https://github.com/omigroup/gltf-extensions/discussions/139

**Comment by madjin** [comment]
## Metaverse Interop Community Showcase / Expo

Last week we discussed ideas for showcasing gltf-extensions activities / projects as booths sorta like a living blog or knowledge space
![brave_FO359Z3x9c](https://user-images.githubusercontent.com/32600939/216471615-51aa0856-0c3f-4ce0-9c54-fb1ae0a6b28a.png)

@antpb is working on booths, here's one he made for personality extension that will later be templatized
https://user-images.githubusercontent.com/32600939/216471691-c423d474-1761-41b5-978f-2fdd74511f91.mp4


Meeting up tomorrow to go over the booth templates: https://hyperfy.io/street
![image](https://user-images.githubusercontent.com/32600939/216471306-3fa04358-ce39-4988-8e1f-7fdbc9f7e2f6.png)

Can download booths here: https://github.com/NEON-BUIDL/booths and via Sketchfab: https://sketchfab.com/m3org/models

Similar to vket this event will be weeks long with opportunities sprinkled in to host panels / talks / presentations
